extensions/guacamole-auth-sso/modules/guacamole-auth-sso-ssl/src/main/resources/guac-manifest.json

```diff
@@ -0,0 +1,35 @@
+{
+
+    "guacamoleVersion" : "1.5.0",
+
+    "name"      : "SSL Authentication Extension",
+    "namespace" : "ssl",
+
+    "authProviders" : [
+        "org.apache.guacamole.auth.ssl.SSLAuthenticationProvider"
+    ],
+
+    "css" : [
+        "styles/sso-providers.css"
```

Comment:
Where is this file? I don't see it anywhere.

Reply:
This, too, is inherited from "guacamole-auth-sso-base": https://github.com/apache/guacamole-client/blob/78c084be6568ae2ae8097daddb1cbf94903df060/extensions/guacamole-auth-sso/modules/guacamole-auth-sso-base/src/main/resources/styles/sso-providers.css